test/commands/passphrase/decrypt.test.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect, test } from '@oclif/test';
+import { cryptography } from 'lisk-elements';
+import * as config from '../../../src/utils/config';
+import * as print from '../../../src/utils/print';
+import * as getInputsFromSources from '../../../src/utils/input';
+
+describe('passphrase:decrypt', () => {
+	const defaultEncryptedPassphrase =
+		'salt=d3887df959ed2bfe5961a6831da6e177&cipherText=1c08a1&iv=096ede534df9092fd4523ec7&tag=2a055e1c860b3ef76084a6c9aca68ce9&version=1';
+	const passphrase = '123';
+	const encryptedPassphraseObject = {
+		salt: 'salt',
+		cipherText: 'cipherText',
+		iv: 'iv',
+		tag: 'tag',
+		version: 1,
+	};
+	const defaultInputs = {
+		password: '456',
+		data: `${defaultEncryptedPassphrase}\nshould not be used`,
+	};
+
+	const printMethodStub = sandbox.stub();
+	const setupTest = () =>
+		test
+			.stub(print, 'default', sandbox.stub().returns(printMethodStub))
+			.stub(config, 'getConfig', sandbox.stub().returns({}))
+			.stub(
+				cryptography,
+				'parseEncryptedPassphrase',
+				sandbox.stub().returns(encryptedPassphraseObject),
+			)
+			.stub(
+				cryptography,
+				'decryptPassphraseWithPassword',
+				sandbox.stub().returns(passphrase),
+			)
+			.stub(
+				getInputsFromSources,
+				'default',
+				sandbox.stub().resolves(defaultInputs),
+			);
+
+	describe('passphrase:decrypt', () => {
+		setupTest()
+			.stdout()
+			.command(['passphrase:decrypt'])
+			.catch(error =>
```

Comment:
Normally we put brackets around expectations even when there's only one for consistency.